src/cascadia/TerminalConnection/ConptyConnection.h

```diff
@@ -41,6 +41,7 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         hstring _startingDirectory;
         hstring _startingTitle;
         guid _guid{}; // A unique session identifier for connected client
+        hstring _clientName{}; // The name of the process hosted by this ConPTY connection (as of launch).
```

Comment:
keeping in case we need it? or can it just be confined to a local

Reply:
I thought we verbally mentioned that this information might be useful under the debugger so I thought "ehhh what the heck". I can move it back to a local if you want.

Reply:
Fair enough